kvui.py

```diff
@@ -921,9 +921,11 @@ def connect_bar_validate(sender):
         hint_panel = self.add_client_tab("Hints", HintLayout(self.hint_log))
         self.log_panels["Hints"] = hint_panel.content
 
-        self.main_area_container = MDGridLayout(size_hint_y=1, cols=1)
-        self.main_area_container.add_widget(self.tabs)
-        self.main_area_container.add_widget(self.screens)
+        self.main_area_container = MDGridLayout(size_hint_y=1, rows=1)
+        tab_container = MDGridLayout(size_hint_y=1, cols=1)
+        tab_container.add_widget(self.tabs)
+        tab_container.add_widget(self.screens)
+        self.main_area_container.add_widget(tab_container)
```

Comment:
@alwaysintreble I don't know what you were trying to do with https://github.com/ArchipelagoMW/Archipelago/pull/4930 , so I don't know how to test it.
Please make sure that what you were trying to do works with this.